imagePNG.ml

```diff
@@ -989,6 +989,21 @@ module ReadPNG : ReadImage = struct
      | _ -> assert false
 end
 
+module type OUT_CHANNEL = sig
```

Comment:
Could you change the indentation to the following (for consistency with the beginning of the file).
```ocaml
module type OUT_CHANNEL = sig
  type out_channel
  val output_string : out_channel -> string -> unit
  val output_char : out_channel -> char -> unit
end

module Buffer_channel = struct
  type out_channel = Buffer.t
  let output_string = Buffer.add_string
  let output_char = Buffer.add_char
end
```

Reply:
I think this module / interface should be defined somewhere else, since it'll likely be useful for the other image formats too.

Reply:
@cfcs you are right, but I was thinking of merging first, and then take care of moving things myself. I want to go over the interface of the library to add the missing documentation stuff.